### PR TITLE
Using Laravel Forge with Linode

### PR DESCRIPTION
**Pull request by dongido001**
This is an article on how to **Use Laravel Forge to deploy your php projects with Linode** guide for the Linode Bounty program.

### CODE REVIEW

**Comment by GuessWhoSamFoo** [comment]
The figures for submission are not up to Linode's standards for publishing. Can you resubmit them with a more visually appealing aesthetic?

**Comment by dongido001** [comment]
Ok

**Comment by dongido001** [comment]
@GuessWhoSamFoo updated

File: docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+## Add SSH keys to your profile.
+You should have received an email already about details of your ssh from above. 
+
+[![Linode API](/docs/assets/configuration-management/adding_public_key.png)](/docs/assets/configuration-management/adding_public_key.png)
+
+Click on the `SSH Keys` and upload your local SSH Key by following the link on the mail or see How to create and copy an SSH Key.  Once you have your SSH key, add a name and your key to the form and submit by clicking on `ADD KEY`
+
+
+## Get your Linode API key
```
Comment (GuessWhoSamFoo) [inline docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md]: Remove this step. Just link existing Linode guide: https://www.linode.com/docs/platform/api/api-key

File: docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+1. Sign up for a Larvel Forge account if you don't have one ([Laravel Forge](https://forge.laravel.com/auth/register))
+2. Sign up or Log into your Linode account. ([Linode](https://www.linode.com/))
+
+## Link to a Source Control (GitHub, Gitlab or Bitbucket)
+
+If you want to deploy from either of these services( Github, Gitlab or bitbucket), you need to link them to your Forge account. Once you are logged in to your Forge Account, head to your profile tab.
+
+1. Choose your preferred source control by clicking on the `Source Control` Link on the Left.
+
+[![Linode API](/docs/assets/configuration-management/linking_source_control.png)](/docs/assets/configuration-management/linking_source_control.png)
```
Comment (GuessWhoSamFoo) [inline docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md]: Use a red rectangular selection tool per example from another guides: https://www.linode.com/docs/platform/api/api-key

File: docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+SSL (Secure Sockets Layer) is the standard security technology for establishing an encrypted link between a web server and a browser. To add SSL:
+
+1. Choose your server on Forge.
+
+2. Select the domain name that you want to add SSL to. 
+
+3. Click on `SSL` from the left options.
+
+
+[![Linode API](/docs/assets/configuration-management/laravel_forge_adding_ssl.png)](/docs/assets/configuration-management/laravel_forge_adding_ssl.png)
```
Comment (GuessWhoSamFoo) [inline docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md]: Convention for domain name per style guide is `example.com` and `www.example.com`

File: docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+3. Fill in the repository field with your username and repository then select the branch you want to use. Now, click on INSTALL REPOSITORY. Take a closer look how the `Repository` Field is inserted which is  gitlab_username/repository_name_without_.git`. 
+
+eg: 
+
+```
+    dongido/transaction-manager
+
+```
+
+[![Linode API](/docs/assets/configuration-management/git_repository_name.png)](/docs/assets/configuration-management/git_repository_name.png)
```
Comment (GuessWhoSamFoo) [inline docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md]: Remove personal identifying repository name. Update remove yellow highlighted marker. Replace with rectangular selection tool per examples in previous guides: https://www.linode.com/docs/platform/api/api-key

File: docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+[![Linode API](/docs/assets/configuration-management/linode_created.png)](/docs/assets/configuration-management/linode_created.png)
+
+
+## Pointing your domain name
+
+If you intend to visit the site you're setting up via a real domain (versus just testing it by visiting the IP address), you have to add the domain because DNS record changes can take a while to update. 
+
+1. Now head back to Laravel Forge account then click on the active server. 
+
+2. Go to your domain name provider, add an ANAME record for the domain or subdomain, and point it to the public IP address of your server. Your public IP address is as seen in the screenshot below, mine is: `41.33.92.166` That's it!
```
Comment (GuessWhoSamFoo) [inline docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md]: Do not use personal IP address. Substitute with example IP address per style guide: https://github.com/linode/docs/blob/master/docs/linode-writers-formatting-guide.md#example-ip-addresses

File: docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+When the server is active, you will also receive mail from Linode with details of things that has been created.
+
+[![Linode API](/docs/assets/configuration-management/mail_from_linode_showing_details_created.png)](/docs/assets/configuration-management/mail_from_linode_showing_details_created.png)
+
+You will also receive a mail that a public key was added to your gitlab account.
+
+Now, lets confirm that the server is created on Linode also, head over to linode under the linode tab, you should see a node that has been created.
+
+
+[![Linode API](/docs/assets/configuration-management/linode_created.png)](/docs/assets/configuration-management/linode_created.png)
```
Comment (GuessWhoSamFoo) [inline docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md]: Red rectangular selection tool per other review comments. Hide your public IP addresses.

File: docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md [hunk 7]
```diff
@@ -0,0 +1,226 @@
...
+
+[![Linode API](/docs/assets/configuration-management/active_servers.png)](/docs/assets/configuration-management/active_servers.png)
+
+When you visit your IP Address, you will see that our server has been set up for us.
+
+[![Linode API](/docs/assets/configuration-management/server_set_up_with_php_7_1.png)](/docs/assets/configuration-management/server_set_up_with_php_7_1.png)
+
+When the server is active, you will also receive mail from Linode with details of things that has been created.
+
+[![Linode API](/docs/assets/configuration-management/mail_from_linode_showing_details_created.png)](/docs/assets/configuration-management/mail_from_linode_showing_details_created.png)
```
Comment (GuessWhoSamFoo) [inline docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md]: Mask private information per other comments with a red selection tool per styling guidelines.

File: docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md [hunk 8]
```diff
@@ -0,0 +1,226 @@
...
+   [![Linode API](/docs/assets/configuration-management/server_credential.png)](/docs/assets/configuration-management/server_credential.png)
+
+When you click on `create server`, you will get a pop up showing your sudo password and database password. There are the details of the database that has been created for you.
+
+Now, your newly created server will be listed in the Active server below. You should now see that Forge is going through the paces of creating a server, configuring it, and getting it ready based on the settings we just input.
+
+Once the new server is ready to go, you will see it's green and good to go(By default, it takes some seconds).
+
+
+[![Linode API](/docs/assets/configuration-management/active_servers.png)](/docs/assets/configuration-management/active_servers.png)
```
Comment (GuessWhoSamFoo) [inline docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md]: Mask public IP address. Use red selection tool per other review comments.

File: docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md [hunk 9]
```diff
@@ -0,0 +1,226 @@
...
+
+**Region** Wherever you want your server to be located. Ideally you'd want it to be close to where you believe your main users will be.
+
+**PHP Version** This will be the php version that will be installed on your Linode server.
+
+**Database Name** If your application needs a database, then you can name it here. By default it will be named `forge`.
+
+2. Once you have the configuration you like, go ahead and click Create Server. 
+
+   [![Linode API](/docs/assets/configuration-management/server_credential.png)](/docs/assets/configuration-management/server_credential.png)
```
Comment (GuessWhoSamFoo) [inline docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md]: Mask sensitive information using red rectangular selection tool per styling examples.

**Comment by GuessWhoSamFoo** [comment]
@dongido001 Any updates with this?

**Comment by dongido001** [comment]
@GuessWhoSamFoo sorry this might be coming late. My Forge account expired, I had to renew it so I can make changes. 

I've updated the article.

Thanks.

**Comment by GuessWhoSamFoo** [comment]
Closed for internal editing. See PR above for updates.

**Pull request closed without merging**